src/CSET/operators/plot.py

```diff
@@ -656,14 +781,119 @@ def plot_vertical_line_series(
+        # Do the actual plotting.
+        _plot_and_save_vertical_line_series(
+            cube_slice,
+            coord,
+            plot_filename,
+            series_coordinate,
+            title=title,
+            vmin=vmin,
+            vmax=vmax,
+        )
+        plot_index.append(plot_filename)
+
+    # Add list of plots to plot metadata.
+    complete_plot_index = _append_to_plot_index(plot_index)
+
+    # Make a page to display the plots.
+    _make_plot_html_page(complete_plot_index)
+
+    return cube
+
+
+def plot_histogram_series(
+    cube: iris.cube.Cube,
+    filename: str = None,
+    sequence_coordinate: str = "time",
+    stamp_coordinate: str = "realization",
+    **kwargs,
+) -> iris.cube.Cube:
+    """Plot a histogram plot for each vertical level provided.
+
+    A histogram plot can be plotted, but if the sequence_coordinate (i.e. time)
+    is present
+    then a sequence of plots will be produced using the time slider
+    functionality to scroll through histograms against time.
+
+
+    The cube must be 1D.
+
+    Parameters
+    ----------
+    cube: Cube
+        Iris cube of the data to plot. It should have a single dimension.
+    filename: str, optional
+        Name of the plot to write, used as a prefix for plot sequences. Defaults
+        to the recipe name.
+    sequence_coordinate: str, optional
+        Coordinate about which to make a plot sequence. Defaults to ``"time"``.
+        This coordinate must exist in the cube and will be used for the time slider.
+    stamp_coordinate: str, optional
+        Coordinate about which to plot postage stamp plots. Defaults to
+        ``"realization"``.
+
+    Returns
+    -------
+    Cube
+        The original cube (so further operations can be applied).
+
+    Raises
+    ------
+    ValueError
+        If the cube doesn't have the right dimensions.
+    TypeError
+        If the cube isn't a single cube.
+    """
+    recipe_title = get_recipe_metadata().get("title", "Untitled")
+
+    # Ensure we have a name for the plot file.
+    if filename is None:
+        filename = slugify(recipe_title)
+
+    # Ensure we've got a single cube.
+    cube = _check_single_cube(cube)
+
+    # Call internal plotting function
+    plotting_func = _plot_and_save_histogram_series
+
+    # Make postage stamp plots if stamp_coordinate exists and has more than a
+    # single point.
+    try:
+        if cube.coord(stamp_coordinate).shape[0] > 1:
+            plotting_func = _plot_and_save_postage_stamp_histogram_series
+    except iris.exceptions.CoordinateNotFoundError:
+        pass
+
+    # If several histograms are plotted with time as sequence_coordinate
+    # for the time slider option.
+    try:
+        cube.coord(sequence_coordinate)
+    except iris.exceptions.CoordinateNotFoundError as err:
+        raise ValueError(f"Cube must have a {sequence_coordinate} coordinate.") from err
+
+    # set the lower and upper limit for the colorbar to ensure all plots
```

Comment:
Should we be having a colorbar for the histrogram? Doesn't this refer to the limits for one of the axis rather than colorbar in the comment?

Reply:
![image](https://github.com/user-attachments/assets/050bf7bf-fea1-406c-9895-294e3193672b)

Reply:
Thanks, i removed the colourbar comment